contracts/child/modules/CVSStorage.sol

```diff
@@ -46,4 +44,14 @@ abstract contract CVSStorage is ICVSStorage {
     function getValidator(address validator) public view returns (Validator memory) {
         return _validators.get(validator);
     }
+
+    function verifySignature(address signer, uint256[2] calldata signature, uint256[4] calldata pubkey) internal view {
```

Comment:
can we rename this to something else, like `verifyValidatorRegistration()` so that it's not mixed up for a function that does BLS signature verifications?

Reply:
done